int32 n
time stamp
string type
geometry_msgs/Point32 point
geometry_msgs/Point32 scale
float32 confidence
